taskcluster/scripts/tc-admin-apply-staging.py

```diff
@@ -13,20 +13,29 @@
 assert "TASKCLUSTER_PROXY_URL" in os.environ
 options = {"rootUrl": os.environ["TASKCLUSTER_PROXY_URL"]}
 auth = taskcluster.Auth(options)
+secrets = taskcluster.Secrets(options)
 
 result = auth.resetAccessToken(CLIENT_ID)
 assert result
 
 token = result["accessToken"]
 assert isinstance(token, str)
 
+result = secrets.get("project/releng/tc-github-private-key")
+assert isinstance(result, dict)
+
+github_app_creds = result["secret"]
+assert isinstance(github_app_creds, dict)
+
 env = os.environ.copy()
 del env["TASKCLUSTER_PROXY_URL"]  # ensure we don't use the proxy client
 env.update(
     {
         "TASKCLUSTER_ROOT_URL": ROOT_URL,
         "TASKCLUSTER_CLIENT_ID": CLIENT_ID,
         "TASKCLUSTER_ACCESS_TOKEN": token,
+        "GITHUB_APP_ID": str(github_app_creds["app_id"]),
+        "GITHUB_APP_PRIVKEY": github_app_creds["ssh_privkey"],
```

Comment:
Is there any chance of this leaking through the env? (Do we even care for staging?